wordpress-mobile/WordPress-Android
# Promotion Dialog for Aztec

### Fix
Add new editor promotion dialog code back for use when Aztec is published to open beta as described in https://github.com/wordpress-mobile/WordPress-Android/issues/5834.  These changes were tested on all API's supported to ensure they don't cause [this crash](https://github.com/wordpress-mobile/WordPress-Android/issues/5787).

![dialog](https://cloud.githubusercontent.com/assets/3827611/25717555/085dca00-30c0-11e7-8bd1-057e54fe1138.png)

### Test
1. Go to ***Sites*** tab.
2. Notice new editor promotion dialog.
3. Dismiss dialog with ***OK*** or back button.
4. Go to another tab.
5. Go to ***Sites*** tab.
6. Notice no dialog is shown.

#### Note
The promotion dialog was also updated to use the style of other dialogs in the app for consistency.  That code is shared with Stats and is used [here](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java#L640).  See the screenshots below for examples.

![dialogs](https://cloud.githubusercontent.com/assets/3827611/25717451/bb8ccee2-30bf-11e7-9a03-101840534a81.png)
